### ISSUE TITLE
Tips for Tensorflow 2.0 NER task by using fit method.

### ISSUE THREAD

**Issue by aiscientist** [huggingface/transformers#6209]
# ❓ Questions & Help

## Details

Any tips for implementing TF2 NER (TF Token Classifier) by using model.fit method?

An official example does not use model.fit for NER task.

Any links or tips will be appreciated.

**A link to original question on the forum/Stack Overflow**:

**Comment by TarasPriadka** [comment]
@aiscientist 
Why would you need to use `model.fit()` method? I think it is a more low level way to train your model. There's an easy way to train HuggingFace models with `TFTrainer`. All you need to do, is to initialize pretrained config, tokenizer and model, create your own Dataset class for NER data, and just pass it to the trainer. It will do all the rest.

**Comment by aiscientist** [comment]
> @aiscientist
> Why would you need to use `model.fit()` method? I think it is a more low level way to train your model. There's an easy way to train HuggingFace models with `TFTrainer`. All you need to do, is to initialize pretrained config, tokenizer and model, create your own Dataset class for NER data, and just pass it to the trainer. It will do all the rest.

Yes, I need to finish this job using model.fit() following from the TensorFlow guideline (Like Keras Style), but for NER task it's kinda hard to manage loss because of model.fit().  Using Pytorch is easier but, I need to use model.fit for this task because of several reasons. I just want to know that it is possible to use TF Token Classifier on model.fit(). If not, I need to discuss with my co-workers and find another way to do it.
